pkg/backend/display/ai.go

```diff
@@ -0,0 +1,238 @@
+package display
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+
+	pkgWorkspace "github.com/pulumi/pulumi/pkg/v3/workspace"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/env"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/workspace"
+)
+
+var atlasBaseURLs = map[string]string{
+	"https://api.pulumi.com": "https://app.pulumi.com",
+
+	// FIXME(dev): Remove before merging
+	// Local instance logged into using `pulumi login https://api.simon-local.pulumi.local`
+	"https://api.simon-local.pulumi.local": "https://app.simon-local.pulumi.local",
```

Comment:
You can use this for now: https://app.pulumi.com/pulumi-ai/atlas/api/ai/chat/preview
We will remap it to "api.pulumi" but this Url will likely be stable.
Might want to add "X-Pulumi-Origin" in the request - this is used to "hide" it from the public.